.github/workflows/build.yml

```diff
@@ -0,0 +1,40 @@
+# build workflow builds docker images, pushes images to docker hub and updates swagger API
+on:
+  push:
+    branches: [develop, master]
+name: Build
+jobs:
+  build:
+    strategy:
+      matrix:
+        go-version: [1.15.x]
+        os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+    env:
+      GOPATH: ${{ github.workspace }}
+      SWAGGER_API_KEY: ${{ secrets.swagger_api_key }}
```

Comment:
How are these secrets configured?

Reply:
github secrets. they are decrypted transparently only pushed to develop or master

Reply:
they should be present under project settings